pkgs/development/python-modules/scramp/default.nix

```diff
@@ -40,6 +42,7 @@ buildPythonPackage rec {
 
   postPatch = ''
     # Upstream uses versioningit to set the version
+    sed -i "/versioningit >=/d" pyproject.toml
     sed -i '/^name =.*/a version = "${version}"' pyproject.toml
     sed -i "/dynamic =/d" pyproject.toml
```

Comment:
nit: those could have been combined into one command

Reply:
Yes, you're right. At the same time, even though it's less efficient, I feel that keeping separate commands here is more readable. But this motivates me to look into whether we can handle versioningit better in general, thank you!

Reply:
I always find less code overhault more readable unless you are using some special character shortcuts. Like here, eg deduplicating pyproject.toml but as said it is not that super important and focusing on getting a setup-hook for verioningit is a better solution anyway.